py/torch_tensorrt/dynamo/_defaults.py

```diff
@@ -31,8 +32,17 @@
 DRYRUN = False
 HARDWARE_COMPATIBLE = False
 SUPPORTED_KERNEL_PRECISIONS = {dtype.f32, dtype.f16, dtype.bf16, dtype.i8, dtype.f8}
-TIMING_CACHE_PATH = os.path.join(tempfile.gettempdir(), "timing_cache.bin")
+TIMING_CACHE_PATH = os.path.join(
+    tempfile.gettempdir(), "torch_tensorrt_engine_cache", "timing_cache.bin"
+)
 LAZY_ENGINE_INIT = False
+SAVE_ENGINE_CACHE = True
+LOAD_ENGINE_CACHE = True
+ENGINE_CACHE_DIR = os.path.join(tempfile.gettempdir(), "torch_tensorrt_engine_cache")
+ENGINE_CACHE_SIZE = 1073741824
+ENGINE_CACHE_INSTANCE = EngineCache(
```

Comment:
I think we can leave this as `None` as default and just create the cache object in compile

Reply:
This is because if users run `compile` multiple times, the EngineCache class will be instantiated multiple times. e.g.
```
for i in range(3):
    torch.compile(...)
```

Please let me know if I'm wrong or there's better way to bypass the issue

Reply:
But a cache object is just an interface to a system or process outside of compile right? like we can have as many EngineCache objects as we want since they all ultimately hit the disk

Reply:
yes but if that's created too many times. Do the instances use too much memory?

Reply:
Do you want to create multiple EngineCache objects or just single one?